frameworks/hdfs/tools/docker-client/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:16.04
+
+RUN apt-get -y update && \
```

Comment:
How much size is saved by condensing these three `RUN`s into a single `RUN`?

Reply:
#layers

Reply:
It's not really in consideration of size. It's just best practice to have `update` in the same layer as other `apt-get` runs so as to not cache the `update` layer and reference older packages if at a later the following `apt-get`s are modified in their own layers.

Reply:
Would it not be better to run all `apt` commands once? Finishing off with a cleanup? See https://github.com/mesosphere/dcos-tensorflow/blob/master/images/Dockerfile#L10